packages/orchestrator-ui-components/src/components/WFOWorkflowSteps/WFOStepList/WFOStepList.tsx

```diff
@@ -36,10 +41,17 @@ export const WFOStepList = React.forwardRef(
         let stepStartTime = startedAt;
 
         useImperativeHandle(reference, () => ({
-            scrollToStep: (stepId: string) =>
+            scrollToStep: (stepId: string) => {
                 stepReferences.current.get(stepId)?.scrollIntoView({
                     behavior: 'smooth',
-                }),
+                });
+                const foundStepListItem = stepListItems.find(
+                    (value) => value.step.stepId === stepId,
+                );
+                if (foundStepListItem) {
+                    onExpandStepListItem(foundStepListItem);
```

Comment:
This functions isn't triggered when the item is expanded but it's actually expading it so a better namen would be expandStepListItem or doExpandStepListItem if you want to keep using verbs.

Reply:
The component "WFOStepList" exposes this function as a prop. Therefore, by convention, we should name it on..... (similar to onClick etc)

Reply:
rename to triggerExpand....

Reply:
renamed to onTrigger...